README.md

```diff
@@ -18,62 +18,36 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-databricks) to se
         <img src="https://img.shields.io/badge/discourse-browse_forum-red.svg?color=0052FF&labelColor=090422&logo=discourse" /></a>
 </p>
 
-## Welcome!
-
-Prefect integrations for interacting with Databricks
-
-The tasks within this collection were created by a code generator using the service's OpenAPI spec.
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-databricks) to see additional examples and the API reference.
 
-The service's REST API documentation can be found [here](https://docs.databricks.com/dev-tools/api/latest/index.html).
+The prefect-databricks collection makes it easy to create, run, and manage Databricks jobs in your Prefect flows. Check out the examples below to get started!
```

Comment:
I think coordinating Databricks jobs with other tools is the value prop here, since that's the part that's difficult to do with only Databricks.
```suggestion
The prefect-databricks collection makes it easy to coordiante Databricks jobs with other tools in your data stack using Prefect. Check out the examples below to get started!
```